### PR TITLE
Hyundai: Add FW for 2023 Ioniq 5 HDA2

### PR DESCRIPTION
**Pull request by sunnyhaibin**
Add firmware for the 2023 Hyundai Ioniq 5 HDA2. Expand supported model-year range.

Route ID: `8a53aa70d88313f1|2022-11-01--15-17-19`

Thanks to community Ioniq 5 owner jdtram.

### CODE REVIEW

File: docs/CARS.md [hunk 1]
```diff
@@ -6,216 +6,216 @@ A supported vehicle is one that just works when you install a comma three. All s
 
 # 208 Supported Cars
 
-|Make|Model|Supported Package|ACC|No ACC accel below|No ALC below|Steering Torque|Resume from stop|Harness|
```
Comment (sshane) [inline docs/CARS.md:9]: are you just running ./selfdrive/cars/docs.py? make sure your IDE doesn't touch this file

**Comment by sunnyhaibin** [comment]
The route showed that openpilot recognized the vehicle and was able to engage/disengage, but the user mentioned that there was no steering or indication that openpilot was initialized. The dashboard of the vehicle had different errors/alerts shown.

When "Experiental openpilot longitudinal control" was enabled, openpilot returned with "Harness Relay Malfunction". When the toggle was disabled, no error was shown on screen but still with errors/alerts on the screen.

User mentioned that he had swapped out all the hardware excluding the comma three and the Red Panda.
* `8a53aa70d88313f1|2022-10-19--08-00-55`

@sshane Is there anything else the user should try out?

image
image

**Comment by sshane** [comment]
@adeebshihadeh any idea what's going on or should I take a look?

**Comment by sunnyhaibin** [comment]
The user is obtaining a new Red Panda to verify if it is the root cause of the issue. Will update when the user tests with the new Red Panda. The old Red Panda is being sent back to comma for diagnosis.

**Comment by sunnyhaibin** [comment]
User reported that the issue is resolved with a new Red Panda.

**Pull request merged**